### ISSUE TITLE
Braille Settings Panel Checkbox Overlap Issue in NVDA 2024.4beta2

### ISSUE THREAD

**Issue by nishimotz** [nvaccess/nvda#17141]
### Steps to reproduce:

1. Use NVDA 2024.4beta2 (portable version) on a Japanese Windows 11 system.
2. Set the NVDA interface language to English.
3. Start NVDA.
4. Open the "Settings" dialog.
5. Navigate to the "Braille" panel.

### Actual behavior:

The "Speak character when routing cursor in text" checkbox is displayed below the "Paragraph start marker" combo box and overlaps with other combo boxes.

### Expected behavior:

The "Speak character when routing cursor in text" checkbox should be displayed below the "Formatting display" setting, according to the logical tab order.

### NVDA logs, crash dumps and other attachments:

```
INFO - __main__ (14:01:42.882) - MainThread (1344):
Starting NVDA version 2024.4beta2 x86
INFO - core.main (14:01:43.133) - MainThread (1344):
Config dir: f:\nvda\portables\beta\userConfig
INFO - config.ConfigManager._loadConfig (14:01:43.133) - MainThread (1344):
Loading config: f:\nvda\portables\beta\userConfig\nvda.ini
INFO - core.main (14:01:43.164) - MainThread (1344):
Developer Scratchpad mode enabled
INFO - core.main (14:01:43.742) - MainThread (1344):
Windows version: Windows 11 23H2 (10.0.22631) workstation AMD64
INFO - core.main (14:01:43.742) - MainThread (1344):
Using Python version 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:00:00) [MSC v.1938 32 bit (Intel)]
```

#### NVDA installed/portable/running from source:
Portable

#### NVDA version:
2024.4beta2

#### Windows version:
Windows 11 23H2 (10.0.22631.4112)

#### Name and version of other software in use when reproducing the issue:
None

#### Other information about your system:
None

#### Does the issue still occur after restarting your computer?
Yes, the issue occurs regardless of restarting the computer.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, NVDA version 2024.3.1 does not have this display issue in the settings panel.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes, the problem occurs even when NVDA add-ons are disabled.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes, the problem still occurs after running the COM Registration Fixing Tool.

![image](https://github.com/user-attachments/assets/1ff2cc6c-ff09-46e2-a2e6-371cf01e012b)

**Comment by SaschaCowley** [comment]
This is reproducible on English systems too.
